### ISSUE TITLE
Blink Dagger cooldown text is colored RED and is unreadable!

### ISSUE THREAD

**Issue by vozaldi** [ValveSoftware/Dota2-Gameplay#8177]
### Item name

Blink Dagger

### Description

Hello,

I bought Blink Dagger as first item. When I got attacked it shows cooldown "3, 2, 1" in red colored text, which unreadable with Blink Dagger's dark colored icon.

Please solve this issue as it's really hard to see the cooldown text (screenshot attached).

![image](https://user-images.githubusercontent.com/10706082/233663646-eba96436-d058-4586-8e5d-115ec80ceb15.png)

### Example Match ID and Timestamp

7117583575@10:00

**Comment by 00bugs** [comment]
Exactly!!! it's a small issue that needs to be resolved cause when I try to shift focus to check the remaining cooldown, my vision gets extremely confused, especially in wars. So, please resolve this tiny snag as soon as you can please!

**Comment by adihsd** [comment]
please fix this, its incredibly bad for some some reason.

**Comment by Tamal-Debnath** [comment]
adding on to this with what someone else commented earlier, 
"I uninstalled Dota for this very reason and I know what the reason is. I have migraine since I was 8 years old. Migraineurs have pre-symptoms, one of which is a visual aura that bears an exact resemblance to this Dagger issue. This visual aura is extremely annoying for migraineurs (it is very scary for me. Seeing it implies that I will suffer from a terrible headache for the next few hours). The aura disappears one space in your field of vision, preventing you from reading. Hopefully Valve will change this little inconvenience. At the moment I will not play again in a few months."

Please change the color to something slightly different, its really bad for us migraineurs.

**Comment by ikcis** [comment]
please fix this asap !!!
